qpool benchmarking

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #29274 qadaptive_avgpool2d benchmarking
* #29268 qavgpool benchmarking
* #29257 Adding short tests to qmaxpool benchmarking
* **#29250 qpool benchmarking**

Differential Revision: [D18339142](https://our.internmc.facebook.com/intern/diff/D18339142)

Commit:
qpool benchmarking

Commit:
Update on "qpool benchmarking"


Differential Revision: [D18339142](https://our.internmc.facebook.com/intern/diff/D18339142)

Commit:
Update on "qpool benchmarking"


Differential Revision: [D18339142](https://our.internmc.facebook.com/intern/diff/D18339142)

Commit:
Update on "qpool benchmarking"


Differential Revision: [D18339142](https://our.internmc.facebook.com/intern/diff/D18339142)

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qpool_test.py:
```diff
@@ -0,0 +1,87 @@
+        'contig': (False, True),
+        'dtype': (torch.qint32, torch.qint8, torch.quint8),
+    },
+    tags=('long',)
```
Consider marking it as `short` which is the default mode to avoid tag_filter.

z-a-f replied:
Oh, I thought long and short run on different machines. My long tests are REALLY long -- there are like 7000 combinations for the benchmark parrameters in some of them :) Should I still mark it as short?

mingzhe09088 replied:
long and short will run on the same machine. 7000 combinations seem too much. Right now, each test(a single test of an operator) runs around 10 seconds. With 7000 combinations in one operator, it will take around 7000 * 5 second / 3600 = 9 hours. In our current testing, we are aiming to run all quantized ops within 8 hours. Could you take a look at this and those other tests try to fit that 8 hours time frame? You may want to split the input shapes into short and long. In that case, quick perf verification is possible.

Commit:
Update on "qpool benchmarking"


Differential Revision: [D18339142](https://our.internmc.facebook.com/intern/diff/D18339142)

Commit:
Update on "qpool benchmarking"


Differential Revision: [D18339142](https://our.internmc.facebook.com/intern/diff/D18339142)

Commit:
Update on "qpool benchmarking"


Differential Revision: [D18339142](https://our.internmc.facebook.com/intern/diff/D18339142)

Commit:
Update on "qpool benchmarking"


Differential Revision: [D18339142](https://our.internmc.facebook.com/intern/diff/D18339142)

jianyuh reviewed (approved):
Looks good.

Commit:
Rebasing on "qpool benchmarking"


Differential Revision: [D18339142](https://our.internmc.facebook.com/intern/diff/D18339142)
